Marlin/src/lcd/dogm/u8g_dev_tft_320x240_upscale_from_128x64.cpp

```diff
@@ -214,76 +224,97 @@ static const uint8_t ili9341_init_sequence[] = { // 0x9341 - ILI9341
-     B10000000,B00000000,B00000000,B00000000,B00000001,
-     B10000000,B00000000,B00000000,B00000000,B00000001,
-     B10000000,B00000000,B00000000,B00000000,B00000001,
-     B10000000,B00000000,B00000000,B00000000,B00000001,
-     B10000000,B00000000,B00000000,B00000000,B00000001,
-     B10000000,B00000000,B00000001,B11000000,B00000001,
-     B10000000,B00000000,B01000001,B11000000,B00000001,
-     B10000000,B00000000,B11000001,B11000000,B00000001,
-     B10000000,B00000001,B11111111,B11000000,B00000001,
-     B10000000,B00000011,B11111111,B11000000,B00000001,
-     B10000000,B00000001,B11111111,B11000000,B00000001,
-     B10000000,B00000000,B11000000,B00000000,B00000001,
-     B10000000,B00000000,B01000000,B00000000,B00000001,
-     B10000000,B00000000,B00000000,B00000000,B00000001,
-     B10000000,B00000000,B00000000,B00000000,B00000001,
-     B10000000,B00000000,B00000000,B00000000,B00000001,
-     B10000000,B00000000,B00000000,B00000000,B00000001,
-     B10000000,B00000000,B00000000,B00000000,B00000001,
-     B01111111,B11111111,B11111111,B11111111,B11111110,
+     B01111111,B11111111,B11111111,B11111110,
+     B10000000,B00000000,B00000000,B00000001,
+     B10000000,B00000000,B00000000,B00000001,
+     B10000000,B00000000,B00000000,B00000001,
+     B10000000,B00000000,B00000011,B00000001,
+     B10000001,B11000000,B00000011,B00000001,
+     B10000001,B11000000,B00000011,B00000001,
+     B10000001,B11000000,B00000011,B00000001,
+     B10000001,B11000000,B00000011,B00000001,
+     B10000001,B11000000,B00111111,B11110001,
+     B10001111,B11111000,B00111111,B11110001,
+     B10000111,B11110000,B00000011,B00000001,
+     B10000011,B11100000,B00000011,B00000001,
+     B10000001,B11000000,B00000011,B00000001,
+     B10000000,B10000000,B00000011,B00000001,
+     B10000000,B00000000,B00000011,B00000001,
+     B10000000,B00000000,B00000000,B00000001,
+     B10000000,B00000000,B00000000,B00000001,
+     B10000000,B00000000,B00000000,B00000001,
+     B01111111,B11111111,B11111111,B11111110,
   };
 
+  static const uint8_t button3[] = {
+     B01111111,B11111111,B11111111,B11111110,
+     B10000000,B00000000,B00000000,B00000001,
+     B10000000,B00000000,B00000000,B00000001,
+     B10000000,B00000000,B00000000,B00000001,
+     B10000000,B00000000,B00000000,B00000001,
+     B10000000,B00000000,B00011100,B00000001,
+     B10000000,B00000100,B00011100,B00000001,
+     B10000000,B00001100,B00011100,B00000001,
+     B10000000,B00011111,B11111100,B00000001,
+     B10000000,B00111111,B11111100,B00000001,
+     B10000000,B00011111,B11111100,B00000001,
+     B10000000,B00001100,B00000000,B00000001,
+     B10000000,B00000100,B00000000,B00000001,
+     B10000000,B00000000,B00000000,B00000001,
+     B10000000,B00000000,B00000000,B00000001,
+     B10000000,B00000000,B00000000,B00000001,
+     B10000000,B00000000,B00000000,B00000001,
+     B10000000,B00000000,B00000000,B00000001,
+     B01111111,B11111111,B11111111,B11111110,
+  };
```

Comment:
miss a line look like 
![image](https://user-images.githubusercontent.com/145119/63691975-efd11600-c810-11e9-9c54-e11c937e88c3.png)

Reply:
i would keep "color btns left" for the arrows, the other one for ok/cancel